source/reference/kubectl-minio-plugin.rst

```diff
@@ -89,6 +89,71 @@ Subcommands
 - :mc:`~kubectl minio delete`
 - :mc:`~kubectl minio version`
 
+Environment Variables
+---------------------
+
+The :ref:`MinIO Operator <minio-operator-installation>` uses the following environment variables during startup to set configuration settings.
+
+
+.. envvar:: MINIO_OPERATOR_CERTIFICATES_VERSION
+
+   Specifies the certificate API version to use.
+
+   Valid values are ``v1`` or ``v1beta1``.
+
+   When not specified, the default is the API the Kubernetes provides.
+
+.. envvar:: MINIO_OPERATOR_RUNTIME
+
+   Specify the type of runtime to use.
+
+   Valid values are ``EKS``, ``Rancher``, or ``OpenShift``.
+   Leave blank if none of the value options apply.
```

Comment:
I'm not sure what `value options` are, perhaps just `options`?

Reply:
Dropped "value" as suggested. Thanks!